uploader.json

```diff
@@ -1037,7 +1037,27 @@
                 "Received fatal alert: certificate_expired",
                 "has paused due to storage I/O problem"
             ],
-            "symbolic_name": "engine_log"
+            "symbolic_name": "ovirt_engine_log"
```

Comment:
@psachin please revert this change, the symbolic name should be consistent with the existing spec in insights-core.

Reply:
Done